network/src/peer_store/peer_store_impl.rs

```diff
@@ -85,6 +85,16 @@ impl PeerStore {
         ));
     }
 
+    /// Change outbound peer last connected ms
+    pub fn change_outbound_addr_last_connected_ms(&mut self, addr: Multiaddr) {
```

Comment:
please follow the naming convention, rename the fn to `update_xxx`  when updating the struct field:
```
```suggestion
    pub fn update_outbound_addr_last_connected_ms(&mut self, addr: Multiaddr) {
```